numpy/lib/_function_base_impl.pyi

```diff
@@ -316,6 +316,7 @@ def sort_complex(a: ArrayLike) -> NDArray[complexfloating[Any, Any]]: ...
 def trim_zeros(
     filt: _TrimZerosSequence[_T],
     trim: L["f", "b", "fb", "bf"] = ...,
```

Comment:
Do we want to keep supporting `trim=""`? It's not documented or tested for but worked with the previous implementation...